package.json

```diff
@@ -104,5 +105,8 @@
     "build": "node ./scripts/build/build.js",
     "build-docs": "node ./scripts/build-docs.js",
     "check-deps": "node ./scripts/check-deps.js"
+  },
+  "jest-junit": {
```

Comment:
Is it possible to move this to `jest.config.js`?

Reply:
[nope](https://www.npmjs.com/package/jest-junit#configuration). Is there a way to remove it before publishing?

Reply:
Yeah we already modify it in the build script: 
https://github.com/prettier/prettier/blob/3f91d6427d00bb0e247e80575f8c4034c34ca0fc/scripts/build/build.js#L77

Reply:
After re-reading the docs, it looks like `package.json` and environment variables are the only way to configure the package without explicitly enabling it. Putting the config in `jest.config.js` would cause the package to always be loaded, which would cause issues on older mode versions.

Reply:
@palmerj3 is that correct? If so, we should fix it

Reply:
@SimenB I meant that if I put the config in the `reporters` section, that would enable the reporter, which I only want to do on the one test run.

Reply:
That's correct. You can have configuration live in jest.config.js if you want and use the reporter configuration. Most folks have jest-junit loaded as a reporter all the time and just gitignore the junit.xml file since it doesn't add very much overhead at all.

But if you REALLY don't want a junit.xml file generated locally then perhaps use the CI=true environment variable and update your jest.config.js to have a conditional for that.

Reply:
```js
reporters: [
  "default",
  process.env.CI ? ["jest-junit", { ...config }] : null,
].filter(Boolean);
```

or something like that should work, shouldn't it?

Reply:
Yep, exactly.